docs/topics/templates/document-attributes.adoc

```diff
@@ -22,11 +22,11 @@ ifdef::mta[]
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.2.1
+:ProductVersion: 7.0,0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.2.1.GA-redhat-00002
-:ProductDistributionVersion: 6.2.1.GA-redhat
-:ProductDistribution: mta-6.2.1.GA-cli-offline.zip
+:MavenProductVersion: 7.0.0.GA-redhat-00001
```

Comment:
I don't think this is relevant anymore, right? We don't ship anything in Maven

Reply:
Cool, so should I remove [Maven Plugin Guide](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.2/html/maven_plugin_guide/index) for 7.0
Thanks